packages/engine/src/avatar/components/AvatarAnimationComponent.ts

```diff
@@ -158,7 +155,7 @@ export function createVRM(rootEntity: Entity) {
       : formatHumanBones(vrmExtensionDefinition.humanoid!.humanBones as any)
     const bones = humanBonesArray.reduce((bones, bone) => {
       const nodeID = `${documentID}-${bone.node}` as EntityUUID
-      const entity = UUIDComponent.getEntityByUUID(nodeID, layerID)
```

Comment:
Will need to check layer ID with deferred upstream gltf loading, otherwise authoring layer vrm rigs get bound to simulation layer bone entities